<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_PluginAttr_ButtonPart</class>
 <widget class="QWidget" name="P_PluginAttr_ButtonPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>164</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_group" native="true">
     <property name="styleSheet">
      <string notr="true">
QToolButton{
	padding:2px;
	border: 1px solid transparent;
	background: rgba(255,255,255,0);
}
QToolButton:hover{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #fdecad, stop:0.3 #fde48c, stop:0.9 #fce58c, stop:1 #fdfadf )
}
QToolButton:pressed{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}
QToolButton:checked{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton_word">
        <property name="toolTip">
         <string>帮助文档</string>
        </property>
        <property name="text">
         <string>帮助文档</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../drillplugininformationreader.qrc">
          <normaloff>:/DrillPluginInformationReader/Resources/icons/menu/attr_word.png</normaloff>:/DrillPluginInformationReader/Resources/icons/menu/attr_word.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_src">
        <property name="toolTip">
         <string>资源图片</string>
        </property>
        <property name="text">
         <string>资源图片</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../drillplugininformationreader.qrc">
          <normaloff>:/DrillPluginInformationReader/Resources/icons/menu/attr_src.png</normaloff>:/DrillPluginInformationReader/Resources/icons/menu/attr_src.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_highConsumption">
        <property name="toolTip">
         <string>高消耗</string>
        </property>
        <property name="text">
         <string>高消耗</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../drillplugininformationreader.qrc">
          <normaloff>:/DrillPluginInformationReader/Resources/icons/menu/attr_highConsumption.png</normaloff>:/DrillPluginInformationReader/Resources/icons/menu/attr_highConsumption.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_hasTool">
        <property name="toolTip">
         <string>小工具支持</string>
        </property>
        <property name="text">
         <string>小工具支持</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../drillplugininformationreader.qrc">
          <normaloff>:/DrillPluginInformationReader/Resources/icons/menu/attr_hasTool.png</normaloff>:/DrillPluginInformationReader/Resources/icons/menu/attr_hasTool.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../drillplugininformationreader.qrc"/>
 </resources>
 <connections/>
</ui>
